### ISSUE TITLE
Option to ignore the fact that the app is signed with a different certificate

### ISSUE THREAD

**Issue by parasiteoflife** [Droid-ify/client#646]
**Is your feature request related to a problem? Please describe.**
I have some apps installed through the Play Store and those apps are also available on F-Droid and some other repos but currently they can't be updated through Droid-ify because they are signed with different certificates. I also happen to have installed the Core Patch xposed module so this is a non-issue.

**Describe the solution you'd like**
An option to ignore such warnings and try to install the app anyways.

**Describe alternatives you've considered**
I can't uninstall the apps (which are a lot) because I will lose all data on them.
I've considered backing up and restoring the data but that would be such amount of work (because I don't have enough free space to batch backup all the apps in one go) that is really not needed as I can install apps signed with different certificates.

**Additional context**
This is specially needed when you set up your own repo and sign the APKs with your own key, which is my case.

**Comment by realeatham** [comment]
Afaik this isn't possible

**Comment by spider1163** [comment]
Yes it is impossible, it is Android system related, and it's a good thing to avoid malwares

**Comment by Lanchon** [comment]
yes it is possible! this user is stating that he is using the [CorePatch Xposed module](https://github.com/LSPosed/CorePatch) which allows precisely that.

**Comment by ThreeDeeJay** [comment]
Yeah, it's getting increasingly obnoxious how even apps from official sources randomly decide to use a different signature or even package name, usually when they offer variants for GitHub, F-Droid and/or Play Store 🤦
